modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/A11yTool.testcase

```diff
@@ -561,6 +561,34 @@ definition {
 		takeScreenshot();
 	}
 
+	@description = "LPS-142328. Verifies the popup panel should not be present when clicked outside of it."
```

Comment:
Clarify better the state and transition of what we expect:
`Verifies the popup panel will close when click outside of it.`